Samsung-STE: fb hack screen no blank
Samsung-STE: offline charging hack screen no blank
 
diff --git a/minuitwrp/graphics_fbdev.cpp b/minuitwrp/graphics_fbdev.cpp
index 47325d0..cd9e9f7 100644
--- a/minuitwrp/graphics_fbdev.cpp
+++ b/minuitwrp/graphics_fbdev.cpp
@@ -274,8 +274,10 @@ static GRSurface* fbdev_init(minui_backend* backend) {
 
     printf("framebuffer: %d (%d x %d)\n", fb_fd, gr_draw->width, gr_draw->height);
 
+#ifndef TW_NO_SCREEN_BLANK
     fbdev_blank(backend, true);
     fbdev_blank(backend, false);
+#endif
 
     smem_len = fi.smem_len;
 
diff --git a/minui/graphics_fbdev.cpp b/minui/graphics_fbdev.cpp
index eceaeee..eb454c8 100644
--- a/minui/graphics_fbdev.cpp
+++ b/minui/graphics_fbdev.cpp
@@ -152,8 +152,10 @@
 
   printf("framebuffer: %d (%d x %d)\n", fb_fd, gr_draw->width, gr_draw->height);
 
+#ifndef TW_NO_SCREEN_BLANK
   Blank(true);
   Blank(false);
+#endif
 
   return gr_draw;
 }
diff --git a/minuitwrp/graphics_overlay.cpp b/minuitwrp/graphics_overlay.cpp
index 47325d0..0d9e9f7 100644
--- a/minuitwrp/graphics_overlay.cpp
+++ b/minuitwrp/graphics_overlay.cpp
@@ -636,8 +636,10 @@
 
     printf("framebuffer: %d (%d x %d)\n", fb_fd, gr_draw->width, gr_draw->height);
 
+#ifndef TW_NO_SCREEN_BLANK
     overlay_blank(backend, true);
     overlay_blank(backend, false);
+#endif
 
     if (!alloc_ion_mem(frame_size))
         allocate_overlay(fb_fd, gr_framebuffer);
diff --git a/minui/graphics_overlay.cpp b/minui/graphics_overlay.cpp
index eceaeee..1b454c8 100644
--- a/minui/graphics_overlay.cpp
+++ b/minui/graphics_overlay.cpp
@@ -622,8 +622,10 @@
 
     printf("framebuffer: %d (%d x %d)\n", fb_fd, gr_draw->width, gr_draw->height);
 
+#ifndef TW_NO_SCREEN_BLANK
     Blank(true);
     Blank(false);
+#endif
 
     if (!alloc_ion_mem(fi.line_length * vi.yres))
         allocate_overlay(fb_fd, gr_framebuffer);
